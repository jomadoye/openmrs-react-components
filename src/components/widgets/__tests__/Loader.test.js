import React from 'react';
import renderer from 'react-test-renderer';
import Loader from '../Loader';

let props;
let mountedComponent;

const getComponent = () => {
  if (!mountedComponent) {
    mountedComponent = renderer.create(<Loader {...props} />);
  }
  return mountedComponent;
};

describe('Component: Loader', () => {
  beforeEach(() => {
    props = {
    };
    mountedComponent = undefined;
  });

  it('renders properly', () => {
    const component = getComponent();
    expect(component.toJSON()).toMatchSnapshot();
  });
});
