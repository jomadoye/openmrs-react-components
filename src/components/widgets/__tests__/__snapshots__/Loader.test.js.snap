// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Component: Loader renders properly 1`] = `
<div
  className="custom-loader"
>
  <div />
  <div />
  <div />
  <div />
  <div />
  <div />
  <div />
  <div />
  <div />
  <div />
  <div />
  <div />
</div>
`;
